generators/client/templates/angular/src/main/webapp/app/shared/language/_translate-partial-loader.provider.ts

```diff
@@ -31,8 +31,12 @@ export class TranslatePartialLoader implements TranslateLoader {
         return Observable.create(observer => {
             this.http.get(`${this.prefix}/${lang}/${part}${this.suffix}`).subscribe((res) => {
                 let responseObj = res.json();
-                Object.keys(responseObj).forEach(key => {
-                    combinedObject[key] = responseObj[key];
+                Object.keys(responseObj).forEach(key=>{
+                    if(!combinedObject[key]) {
+                        combinedObject[key] = responseObj[key];
+                    }else{
+                        $.extend(true,combinedObject[key], combinedObject[key], responseObj[key])
```

Comment:
wait a minute this should already work as we merge observers in `getPartFile` method and already every state loads 2 files. we need to debug this to why its failing